contracts/cw-proposal-multiple/src/v1_state.rs

```diff
@@ -0,0 +1,93 @@
+//! Here lie all of the types used in v1 proposals, their
```

Comment:
fine to merge this as is (this works fine and is an improvement of over what we have), we'll want to move this to using a dependency on an old version of itself later.

Reply:
Not sure we really need the migration stuff for multichoice because no one is really using it yet as far as I know? Should we remove it?

Reply:
Removed.